### PR TITLE
fix: removed and fixed notice proptypes

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Bug

#### What is expected?

Removed proptypes and fixed warnings.

### CODE REVIEW

File: src/components/Notice/index.tsx [hunk 1]
```diff
@@ -11,8 +10,9 @@ const Container = styled(Box)`
   display: flex;
   align-items: center;
 `
+
 type NoticeProps = {
-  children: ReactNode
+  children: ReactNode | string
```
Comment (chambo-e) [inline src/components/Notice/index.tsx:15]: ?

File: src/components/Notice/index.tsx [hunk 2]
```diff
@@ -11,8 +10,9 @@ const Container = styled(Box)`
   display: flex;
   align-items: center;
 `
+
 type NoticeProps = {
```
Comment (chambo-e) [inline src/components/Notice/index.tsx:14]: Could you add className also pls ? :)

**Review by chambo-e — CHANGES_REQUESTED** [review]

File: src/components/Notice/index.tsx (hunk 1 above)
Comment (matthprost) [reply src/components/Notice/index.tsx:15]: my IDE who's displaying wrong warnings sorry I'm removing it

Comment (matthprost) [reply src/components/Notice/index.tsx:15]: my IDE who's displaying wrong warnings sorry I'm removing it

File: src/components/Notice/index.tsx [hunk 3]
```diff
@@ -11,8 +10,10 @@ const Container = styled(Box)`
...
   children: ReactNode
+  className: string
 } & BoxProps
 
 const Notice = ({ children, ...props }: NoticeProps) => (
```
Comment (chambo-e) [inline src/components/Notice/index.tsx:19]: className need to be passed down here too :)

**Review by chambo-e — APPROVED** [review]

File: src/components/Notice/index.tsx [hunk 4]
```diff
@@ -11,20 +10,19 @@ const Container = styled(Box)`
...
+
 type NoticeProps = {
   children: ReactNode
+  className?: string
 } & BoxProps
 
-const Notice = ({ children, ...props }: NoticeProps) => (
-  <Container {...props}>
+const Notice = ({ children, className, ...props }: NoticeProps) => (
+  <Container className={className} {...props}>
```
Comment (Exayy) [inline src/components/Notice/index.tsx:20]: any reason using for spread everything except className ?

Change not required it was just a question

Comment (matthprost) [reply src/components/Notice/index.tsx:20]: @chambo-e I let you answer haha

Comment (matthprost) [reply src/components/Notice/index.tsx:20]: @chambo-e I let you answer haha

Comment (chambo-e) [reply src/components/Notice/index.tsx:20]: emotion require a type with className to be able to style them, the rest is xstyled props which we will remove soon :)

**Review by vincentaudebert — APPROVED** [review]

**Pull request merged**